### ISSUE TITLE
Cockpit with PHP 7.2

### ISSUE THREAD

**Issue by ronaldaug** [agentejo/cockpit#846]
I think Cockpit still has an issue with PHP 7.2.
I tested on my local machine with PHP 7.2 and it does't work properly.
After the installation, I can't even login and create a new user account.

After that, I tested on shared hosting (online) again and it works totally fine. 
I tried to figure out what happen and found out the php version of shared hosting is 7.1 .

Then, I downgrade my localhost PHP version to 7.1 and now it is working fine.

**Comment by ronaldaug** [comment]
I check my PHP version in both CLI "**php -v**" and .php file "**phpinfo();**"
The version is "7.2.9".

I made a demo video [here](https://www.youtube.com/watch?v=FfgK-apVqeU). 

I fix the first error "**create_function**" in 
`/www/testingcockpit/lib/MongoLite/Database.php`
But it seems there are many errors.

The last error I found is when logging in, this is the error log..

```
Warning: count(): Parameter must be an array or an object that implements Countable in /www/testingcockpit/modules/Cockpit/module/auth.php on line 22
{"success": false}
```

What I can't do is - 
1 - creating a new user, 
2 - accessing account page,
3 - if I logged out, I can't login again.

Here, I use valet server ( but I don't think the error is related to valet because 7.1 works fine )
